doc/cabal-commands.rst

```diff
@@ -607,6 +607,22 @@ you will have access to its quickjump index.
 The generated landing page will contain one tree of all modules per local
 package.
 
+cabal check
+-----------
+
+``cabal check [FLAGS]`` checks the package for common mistakes (e.g.: if
+it is missing important fields like ``synopsis``, if it is using
+tricky GHC options, etc.).
+
+Run ``cabal check`` where your ``.cabal`` package file is.
```

Comment:
I'd somehow add "directory" or "folder" in there

Reply:
good suggestion